packages/rancher-istio/charts/templates/overlay-config-map.yaml

```diff
@@ -5,5 +5,5 @@ metadata:
   name: istio-installer-overlay
   namespace: istio-system
 data:
-  {{ toYaml .Values.overlayFile | indent 2 }}
+  overlay-config.yaml: {{- toYaml .Values.overlayFile | indent 2 }}
```

Comment:
The minus here works ok ? Looks odd to me.

Reply:
I think it will ultimately depend on how the overlayfile is passed in - I can remove it for now

Reply:
it worked, but it does look odd